.eslintrc.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
           'error',
           {
             paths: [
+              {
+                name: 'moment',
+                // tree-shaking for moment is not configured to improve performance - see craco.config.cjs.
+                message: 'moment is not configured for tree-shaking. Is it necessary?',
```

Comment:
Interesting message 🤔 do we need to include it, or can we make it more.. imperative?

Reply:
@zzmp ah i meant the `Is it necessary?` part, is that a default message?